recipes/cgview/meta.yml

```diff
@@ -0,0 +1,39 @@
+{% set name = "cgview" %}
+{% set version = "1.0" %}
+
+about:
+    home: http://wishart.biology.ualberta.ca/cgview/
+    license: GNU General Public License, Version 2.0
+    summary: |
+        CGView is a Java package for generating high quality, zoomable maps of circular genomes. 
+        Its primary purpose is to serve as a component of sequence annotation pipelines, as a
+        means of generating visual output suitable for the web.
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+build:
+    number: 0
+
+source:
+    fn: {{ name }}.zip
+    url: http://wishart.biology.ualberta.ca/cgview/application/{{ name }}.zip
```

Comment:
This does not look versioned.

Reply:
Unfortunately the software is not versioned...there has only ever been one release many years ago and it has never been updated (but still works). I've found a version number (1.0) in one script, but the downloadable tarball is just 'cgview.tar.gz'. Not sure if there is a correct way of handling this...

Reply:
One way to guarantee this is to use: https://github.com/galaxyproject/cargo-port